packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -99,6 +99,20 @@ it('should overflow when there is not enough space', () => {
   expect(breadcrumbs[2].textContent).toEqual('Item 2');
 });
 
+it('should handle overflow when overflowButton is specified', () => {
+  const onClick = jest.fn();
+  useOverflowMock.mockReturnValue([jest.fn(), 2]);
+  const { container } = renderComponent({ overflowButton: onClick });
```

Comment:
Function that returns a `ReactNode` should be passed to `overflowButton`. You can check `ButtonGroup` test.

Reply:
I think I fixed it.